spec/classes/hiera_spec.rb

```diff
@@ -25,13 +25,22 @@
       it { is_expected.to contain_class('hiera::eyaml') }
       it { is_expected.to contain_class('hiera::deep_merge') }
     end
+    describe 'param manage_package => true' do
+      let(:params) do
+        {
+          manage_package: true
+        }
+      end
+
+      it { is_expected.to contain_package('hiera').with('ensure' => 'present') }
+    end
     describe 'param manage_package => false' do
       let(:params) do
         {
           eyaml: true,
           eyaml_gpg: true,
           manage_package: false,
-          keysdir: '/etc/keys'
+          keysdir: '/dev/null/keys'
```

Comment:
is there a reason for changing the parameter here and a few lines below?

Reply:
Yes. For coverage. The same dir "keys" had 3 different names. "So just to use the same dir in all tests, so that they are easier to write. It's very confusing, when the same dir has different paths. I wrote an explanation in the commit, but the commit message drown a bit here.

The coverage percentage gives a more correct measurement of how tested the module is now. The percentage is now around 67% instead of the 30 something, that was reported. Obviously /etc/keys is also just a fake dir and /dev/null van't (easily) be changed. It doesn't come from the code, but most likely from YAML.load(:eyaml). But the path I can't find defined anywhere.